Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM golang:alpine as build
+
+WORKDIR /build
+
+ADD ./ /build/
+ENV CGO_ENABLED=0
+ENV GOOS=linux
+
+RUN apk update && \
+	apk upgrade && \
+	apk add make gcc bash && \
+	rm -rf /var/cache/apk/*
+
+RUN cd ocis && make clean generate build
+
+
+FROM amd64/alpine:latest
```

Comment:
Codacy found an issue: [Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6290476)